config/settings/base.py

```diff
@@ -204,6 +205,12 @@ def env(name, default=NoDefaultValue, type_=str):
     type_=ipv4_networks,
 )
 
+GITHUB_HOOK_SECRET = env(
+    "GITHUB_HOOK_SECRET", default="", type_=lambda x: bytes(x, encoding="utf-8")
+)
+# The ID of the user that GitHub webhook actions should authenticate as:
+GITHUB_USER_ID = env("GITHUB_USER_ID", default=1, type_=int)
```

Comment:
If I want to test in development, do I need to set these env vars? If so, add info to `CONTRIBUTING` and to `env.example`.

Reply:
Also, I think this should be a `str` instead of `int`? It's a hashed field...

Reply:
You _may_ need to set them, depending on what I get resolving the GitHub access problems you spotted.

It's an int, because I don't want its value to depend on the HashID salt. You can _on the Django side_ use the underlying int to look up models.

Reply:
If it's helpful, I was testing with a hashed `id` -- not an int.

Reply:
@wlonk Note that when I was trying to get this to work, I changed it to a string -- feel free to change it back to `int` if that's correct.

Reply:
Resolution: this user is only for local records, and does not need to talk with GitHub at all.

Reply:
Hm, I see it being used in `refresh_commits` to call `get_repo_info` -- which does require GH access?

Reply:
I'm not clear on the implications of saying "this user is only for local records". When trying to perform actions with the `default=1` here, I still get: `metashare.api.models.User.DoesNotExist: User matching query does not exist.` -- so I think we need some docs here for how to add this correctly in local development?

Reply:
Ah, yes, it may need access in `refresh_commits`, let me check that.

As to the 403 issue, I think if you've already auth'd against it, you may need to accept the updated permissions here https://github.com/settings/installations and then re-log-in locally.

Reply:
(If that _does_ fix the issue, I'll document it.)

Reply:
@wlonk I revoked access, re-authorized, and same error. Do you have it working locally?

Reply:
Yes, I do.